### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by agaspardcilia**
I was kinda bored so I updated the readme. I don't now if it's supposed to be in French or English but since it was already in English...

### CODE REVIEW

**Comment by agaspardcilia** [comment]
Also, 4 commits for 29 lines. :sunglasses:

File: README.md [hunk 1]
```diff
@@ -1,15 +1,33 @@
...
+## Installation
+To configure and run you bot, you will need to:
+  - Create app & bot discord: https://discordapp.com/developers/applications/me
+  - Invite him to discord
+  - Create token.js
+  ```js
+  var token = 'BOT TOKEN'
+
+  module.exports = token;
+```
```
Comment (rush93) [inline README.md]: c'est pas mieux d'écrire qu'il faut copier le .dist ?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,33 @@
...
+  module.exports = token;
+``` 
+  - Create youtubeApiKey.js
+  ```js
+  var youtubeApiKey = 'API KEY'
+
+  module.exports = youtubeApiKey;
+```
+  - Install dependencies: `npm i`
+  - Launch the it: `node stipid-econobot.js`
```
Comment (rush93) [inline README.md]: Launch it ?

File: README.md [hunk 3]
```diff
@@ -1,15 +1,33 @@
 # Stupid econobot
-## PREREQUIS
-  - node
+## Prerequisites
+
+To get your bot to work, the following depencies will be required :
+  - node (version?)
```
Comment (rush93) [inline README.md]: 8

**Pull request merged**